api/nntrainer-api-common.h

```diff
@@ -151,6 +151,19 @@ typedef enum {
 typedef int (*ml_train_datagen_cb)(float **input, float **label, bool *last,
                                    void *user_data);
 
+/**
+ * @brief Enumeration for the dataset data type of NNTrainer.
```

Comment:
add more comments about the requirements of dataset - all of them are expected to provide input and label

Reply:
I think this is not a good place to add requirements, maybe dataset handle? Apart from it, I am planning to enhance tizen documentation regarding this topic.

Reply:
Which ever place seems good to you.